docs/src/main/asciidoc/spring-web-guide.adoc

```diff
@@ -45,113 +45,14 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
 ----
 
 This command generates a Maven project with a REST endpoint and imports the `spring-web` extension.
-
-
-== Writing the GreetingController
-
-Before writing the `GreetingController`, let's first create a `Greeting` class that will be used as a response from the controller.
-Furthermore let's create a Spring bean called `GreetingBean` that will be used by the controller.
-
-[source,java]
-----
-package org.acme.spring.web;
-
-public class Greeting {
-
-    private final String message;
-
-    public Greeting(String message) {
-        this.message = message;
-    }
-
-    public String getMessage() {
-        return message;
-    }
-}
-----
-
-
-[source,java]
-----
-package org.acme.spring.web;
-
-import org.springframework.stereotype.Service;
-
-@Service
-public class GreetingBean {
-
-    public String greet(String input) {
-        return "HELLO " + input.toUpperCase() + "!";
-    }
-}
-----
-
-The Quarkus maven plugin automatically generated a `src/main/java/org/acme/spring/web/GreetingController.java` file.
-However since we are going to be using Spring Web annotations to define our REST endpoint (instead of the JAX-RS ones used by default), we need to change
-the `GreetingController` to have the following content:
-
-[source,java]
-----
-package org.acme.spring.web;
-
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.bind.annotation.PathVariable;
-
-@RestController
-@RequestMapping("/greeting")
-public class GreetingController {
-
-    private final GreetingBean greetingBean;
-
-    public GreetingController(GreetingBean greetingBean) {
-        this.greetingBean = greetingBean;
-    }
-
-    @GetMapping("/{name}")
-    public Greeting hello(@PathVariable(name = "name") String name) {
-        return new Greeting(greetingBean.greet(name));
-    }
-}
-----
-
-== Update the test
-
-We also need to update the functional test to reflect the changes made to the endpoint. Change its content to:
-
-
-[source, java]
-----
-package org.acme.spring.web;
-
-import io.quarkus.test.junit.QuarkusTest;
-import org.junit.jupiter.api.Test;
-
-import static io.restassured.RestAssured.given;
-import static org.hamcrest.CoreMatchers.is;
-
-@QuarkusTest
-public class GreetingControllerTest {
-
-    @Test
-    public void testGreeting() {
-        given()
-            .when().get("/greeting/world")
-            .then()
-                .statusCode(200)
-                .body("message", is("HELLO WORLD!"));
-    }
-
-}
-----
+The Quarkus maven plugin automatically generated a src/main/java/org/acme/spring/web/GreetingController.java file with the Spring Web annotations to define our REST endpoint (instead of the JAX-RS ones used by default)
```

Comment:
One additional comment: I think we would want to show the file and make an update to it to show what it looks like. Otherwise, you don't show anything to the user and that's IMHO the whole point of this.

So your sentence is OK but I add instructions about updating the source file (and the test) with the code we had before. That way, the user will be able to see how it looks like and see more Spring Web features.

I think the fact that we pass a parameter with `@PathVariable` should be kept.

Reply:
Ok, I'll modify it for keeping and show the code as you suggest.